README.md

```diff
@@ -42,8 +41,9 @@ To reproduce the results of the analysis, the modules should be run in order.
 | [5.analyze-stability](5.analyze-stability/) | Determine how stable compression solutions are between and across algorithms, and across dimensions |
 | [6.biobombe-projection](6.biobombe-projection/) | Apply BioBombe matrix interpretation analysis and overrepresentation analyses to assign biological knowledge to compression features |
 | [7.analyze-coverage](7.analyze-coverage/) | Determine the coverage, or proportion, of enriched gene sets in compressed latent space features for all models and ensembles of models |
-| [8.gtex-interpret](9.gtex-interpret/) | Interpret compressed features in the GTEX data |
+| [8.gtex-interpret](8.gtex-interpret/) | Interpret compressed features in the GTEX data |
 | [9.tcga-classify](9.tcga-classify/) | Input compressed features from TCGA data into supervised machine learning classifiers to detect pathway aberration |
+| [10.gene-expression-signatures](10.gene-expression-signatures/) | Identify gene expression signatures for sample sex in GTEx and TCGA data, and MYCN amplification in TARGET data |
```

Comment:
This is SO organized!!!  So you don't have a bash script to run the entire pipeline?

Reply:
> So you don't have a bash script to run the entire pipeline?

I do! Thanks for reminding me. I will update it in the next commit!